neo/Network/P2P/Payloads/DisconnectPayload.cs

```diff
@@ -0,0 +1,53 @@
+using Neo.IO;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public enum DisconnectReason : byte
+    {
+        MaxConnectionReached = 0x01,
+        MaxConnectionPerAddressReached = 0x02,
+        DuplicateConnection = 0x03,
+        MagicNumberIncompatible = 0x04,
+        ConnectionTimeout = 0x05,
+        UntrustedIpAddresses = 0x06,
+
+        FormatExcpetion = 0x10,
+        InternalError = 0x11,
+    }
+
+    public class DisconnectPayload : ISerializable
+    {
+        public const int MaxDataSize = 5120;
+
+        public DisconnectReason Reason;
+        public string Message;
```

Comment:
Do we need the message?

Reply:
I think we need, `DisconnectReason` just like error code, we need the details sometimes.

Reply:
Where ? maybe it could be solved adding more entries to the enum

Reply:
I think that the message is not needed, and the protocol should be language independent

Reply:
fixed